Bottom: b1c88a177efd23299460ad58630daf7d7aa0a243
Top:    a4d66316d94b5b89de79b057786b75a1cf0e19ba
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-22 23:59:39 +0000

Refresh of fixup-some-bugs-from-the-added

---

diff --git a/src/misc.c b/src/misc.c
index f60639e..37aaebb 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -2015,7 +2015,6 @@ MoveLayerToGroup (int layer, int group)
 {
   int prev, i, j;
 
-#warning BUG NEXT LINE?
   if (layer < 0 || layer >= max_copper_layer + 2)
     return -1;
   prev = GetLayerGroupNumberByNumber (layer);
